Projects/Contracts/Sources/Contracts.swift

```diff
@@ -85,6 +35,7 @@ extension Contracts: View {
             fetch()
         }
         .trackOnAppear(hAnalyticsEvent.screenView(screen: .insurances))
+        .navigationBarTitle(L10n.InsurancesTab.title)
```

Comment:
This one usually leads to the title being set too late and it looks off, why did you move this away from the Journey?

Reply:
Can move it back, originally I was going to reuse the `ContractsTable` for `Terminated` as well but now I created a new view. So I can just move it back 👯